app/services/reporting/account_reuse_report.rb

```diff
@@ -9,20 +9,51 @@ def initialize(report_date = Time.zone.today)
     # Return array of arrays
     def account_reuse_report
       account_reuse_table = []
-      account_reuse_table << ['Num. SPs', 'Num. users', 'Percentage']
+      account_reuse_table << [
+        'Metric',
+        'Num. all users',
+        '% of accounts',
+        'Num. IDV users',
+        '% of accounts',
+      ]
+
+      total_reuse_report[:sp_reuse_stats].each do |result_entry|
+        account_reuse_table << [
+          "#{result_entry['num_sps']} apps",
+          result_entry['num_all_users'],
+          result_entry['all_percent'],
+          result_entry['num_idv_users'],
+          result_entry['idv_percent'],
+        ]
+      end
+
+      account_reuse_table << [
+        '2+ apps',
+        total_reuse_report[:sp_reuse_stats][:total_all_users],
+        total_reuse_report[:sp_reuse_stats][:total_all_percent],
+        total_reuse_report[:sp_reuse_stats][:total_idv_users],
+        total_reuse_report[:sp_reuse_stats][:total_idv_percent],
+      ]
 
-      total_reuse_report[:reuse_stats].each do |result_entry|
+      # Blank line to separate multiple app use from multiple agency use
+      account_reuse_table << ['', '', '', '', '']
+
```

Comment:
```suggestion
```
let's remove this blank line? it looks funny in the screenshots to me, I think the first column labels are clear enough and easy to differentiate

Reply:
Yeah, it looks funny now, but I was thinking of when there are several lines for each (apps and agencies - i.e. 2 apps, 3 apps, 4 apps, 5 apps, 6 apps, 2+ apps, 2 agencies, 3 agencies, 4 agencies, 5 agencies, 2+ agencies). 

I can still remove it, if you still don't think it makes sense with that context?

Reply:
I think it'll be ok without, worst case we could split into separate tables?